src/common-middleware.ts

```diff
@@ -157,4 +158,5 @@ export default function commonMiddlewareSetup(app) {
    * global router.
    */
   app.get("/artist/*", artistMiddleware)
+  app.get("/search*", searchMiddleware)
```

Comment:
Since [I think] this is scoped to the search route, could it just happen within that app instead of common middleware?

Reply:
Since it appears to be doing a redirect having it in a middleware would be the more performant solution.